TypeCobol/Compiler/CodeElements/Expressions/CallParameter.cs

```diff
@@ -160,4 +160,14 @@ public enum ParameterPassingDirection
         Output,
         Returning
     }
+
+    /// <summary>
+    /// A keyword shall be specified in accordance with the intrinsic function definition.
+    /// For instance the keywords LEADING and TRAILING may be specified as the second and optional argument of TRIM.
+    /// </summary>
+    public enum IntrinsicFunctionKeywordArgument
```

Comment:
I'm not ok with the naming. `IntrinsicFunctionKeywordArgument` is related to the technical aspect of our parser. I'd rather go with `TrimDirection`

Reply:
As discussed, we keep it as it is (those terms are used by IBM).
But I've added a comment on the enumeration items to point out that the keywords are currently only allowed for TRIM.